confidential/account.go

```diff
@@ -0,0 +1,246 @@
+package confidential
+
+import (
+	"crypto/sha256"
+	"github.com/calehh/ristretto255"
```

Comment:
The base crypto lib shoud be created in the repo if we need to  customize it, import from a individual`s repo cause many issue.